quantum/rgb_matrix/rgb_matrix_drivers.c

```diff
@@ -426,11 +426,17 @@ const rgb_matrix_driver_t rgb_matrix_driver = {
 
 // LED color buffer
 LED_TYPE rgb_matrix_ws2812_array[RGB_MATRIX_LED_COUNT];
+bool     updateLEDs = false;
```

Comment:
Jumping from a snake case variable to `updateLEDs` is a little inconsistent.

Should probably be called something like `ws2812_dirty` or `update_required`.